src/components/internal/Description/index.tsx

```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components';
+
+export const Description = styled.span`
+  color: ${({ theme }) => theme.honeycomb.color.bg.disabled};
```

Comment:
```suggestion
  color: ${({ theme }) => theme.honeycomb.color.text.masked};
```

Doesn't that make more sense?